uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/sick-request.controller.js

```diff
@@ -22,26 +22,34 @@ define([
         }
       };
 
+      vm.canSubmit = canSubmit;
```

Comment:
Use `$inject` for all controllers/components touched here.
https://github.com/civicrm/civihr/blob/8fe49c2c7ab52d1d3da0f95dc4c397a0d7ca26f6/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-notification-badge.component.js#L18